### ISSUE TITLE
Can I change the already assigned joystick image?

### ISSUE THREAD

**Issue by Fixid-Fuzz** [RafaelBarbosatec/bonfire#269]
I want to fight while switching characters.
To do this, the joystick action image needs to change when I change the character.
Is it possible to implement this function?

Changing the character was well implemented with the example you gave.

Thank you always for your hard work.

**Comment by RafaelBarbosatec** [comment]
Hi @Fixid-Fuzz !
Thanks for contacting!
You can access the joystick in this way:

```dart
    final joystick = (gameRef.joystick as Joystick);
```

In the moment you can add and remove actions :

```dart
    final joystick = (gameRef.joystick as Joystick);
    joystick.addAction(JoystickAction(actionId: 1));
    joystick.removeAction(1);
```

But not is possible update de `directional`. I will adds this option in next release.

**Comment by Fixid-Fuzz** [comment]
Thank you!!

**Comment by RafaelBarbosatec** [comment]
Available in `2.6.3` version.
```dart
final joystick = (gameRef.joystick as Joystick);
joystick.updateDirectional(JoystickDirectional());
```

**Comment by Fixid-Fuzz** [comment]
I think you are a great creator.
I hadn't even thought of the arrow key sprite problem.
I admire your insight.
thank you
